src/cursos_lic.tex

```diff
@@ -2,13 +2,15 @@
 
 Olá, bixes! Se vocês chegaram até aqui, então parabéns!
 
-Não só porque passaram na FUVEST ou no ENEM, mas também porque entraram na
-Licenciatura em Matemática, mesmo sendo chamados pelos seus colegas de doidos ou
-loucos, entre outros adjetivos simpáticos.
+Não só por terem sido aprovados na FUVEST ou no SISU, mas também porque entraram
+na Licenciatura em Matemática. De antemão, já digo que serão chamados de loucos (e outros
+adjetivos positivos rs), além de ouvir comentários como "Nossa, eu era péssimo 
+em matemática" ou "Tem que gostar bastante, né?" quando dizeram o seu curso para as 
+outras pessoas.
 
 Se vocês ainda não sabem exatamente o que farão com o seu curso, tentaremos
 explicar, mas esperamos que tenham em mente uma coisa: vocês vão ser
-professores, aqueles que sanam as dúvidas dos outros, então foquem em
+\bf{professores}!, aqueles que sanam as dúvidas dos outros, então foquem em
```

Comment:
Eu to relendo essa parte, e tá certo, licenciatura é pra ser professor, mas agora eu fico pensando: "mas poxa, tem tanta coisa que você pode fazer além de dar aula tendo uma licenciatura, como ajudar a escrever livros didáticos, elaboração de problemas, produção de e-books...". Mas acho que mesmo uma pessoa que faz essas coisas e não dá aula continua sendo um professor, né? Não sei haha

Reply:
Eu acho justo esse comentário porque eu também pensei na mesma coisa.  Como já tava no texto, decidi não mudar
Vou mudar para um discurso voltado para "Educação Matemática" ao invés de "ser professor" e vê o que acha ^^